src/algebra/order_functions.lean

```diff
@@ -133,9 +133,13 @@ right_comm max max_comm max_assoc a b c
 lemma max_distrib_of_monotone (hf : monotone f) : f (max a b) = max (f a) (f b) :=
 by cases le_total a b; simp [h, hf h]
 
+alias max_distrib_of_monotone ← monotone.map_max
```

Comment:
I think you could even rename `max_distrib_of_monotone` to `monotone.max_distrib`, your name is clearly better.